src/Bicep.Cli/Services/Evaluation.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Deployments.Core.Definitions.Schema;
+
+namespace Bicep.Cli.Services
+{
+    public class Evaluation
```

Comment:
I would recommend naming this something like `TestEvaluation` to make it clearer that this is related to the test feature.